lib/ace/mode/markdown_worker.js

```diff
@@ -0,0 +1,71 @@
+define(function(require, exports, module) {
```

Comment:
add copyright, author and license notice (similar to how it is done in ACE code)